Hunting Queries/MultipleDataSources/DisabledAccIPSigninWithRareRiskyOps.yaml

```diff
@@ -0,0 +1,91 @@
+id: 9adbd1c3-a4be-44ef-ac2f-503fd25692ee
+name: Successful Signin from IP with Disabled Account - Rare and potentially high-risk Office operations
+description: |
+ 'It is possible that a disabled user account is compromised and another account on the same IP is used to perform operations that are not typical for that user.
```

Comment:
Again you probably want more detail in this description as you are joining a few elements. Think about describing the logic of the query so that someone looking at an incident from this detection knows what it was looking for.